target/linux/octeon/patches-4.19/101-ubnt-split-devicetrees.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/arch/mips/boot/dts/cavium-octeon/ubnt_e100.dts b/arch/mips/boot/dts/cavium-octeon/ubnt_e100-e120.dtsi
+similarity index 92%
+rename from arch/mips/boot/dts/cavium-octeon/ubnt_e100.dts
+rename to arch/mips/boot/dts/cavium-octeon/ubnt_e100-e120.dtsi
+index 243e5dc..000c2d3 100644
+--- a/arch/mips/boot/dts/cavium-octeon/ubnt_e100.dts
++++ b/arch/mips/boot/dts/cavium-octeon/ubnt_e100-e120.dtsi
```

Comment:
I'd make this a little easier by just touching kernel 5.4 and removing all 4.19 changes ...